### PR TITLE
Migrate FAQ from Wiki

### PR DESCRIPTION
**Pull request by hughlang**
Pretty much a straight copy of the wiki FAQ found here: https://github.com/ethereum/wiki/wiki/FAQ

The second level bullets and the inline github links needed to fixed manually. 

I didn't get a chance to fix the length of the section bars, because the script complained. I will do that later.

### CODE REVIEW

File: source/frequently-asked-questions/frequently-asked-questions.rst [hunk 1]
```diff
@@ -2,3 +2,333 @@
...
+Clients
+-------
+
+Where can I find official releases?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  `Releases for
+   AlethZero <https://github.com/ethereum/cpp-ethereum/releases>`__
+-  `Releases for
+   Mist <https://github.com/ethereum/go-ethereum/releases>`__
```
Comment (zelig) [inline source/frequently-asked-questions/frequently-asked-questions.rst]: geth 

and need another entry for mist wallet with correct URL

File: source/frequently-asked-questions/frequently-asked-questions.rst [hunk 2]
```diff
@@ -2,3 +2,333 @@
...
+-  `Releases for
+   Pyethereum <https://github.com/ethereum/pyethereum/releases>`__
+
+How to install development builds?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
```
Comment (zelig) [inline source/frequently-asked-questions/frequently-asked-questions.rst]: again need (only) cross ref to current doc

Comment (hughlang) [inline source/frequently-asked-questions/frequently-asked-questions.rst]: I'll look for those

**Comment by hughlang** [comment]
I know this FAQ content is quite ancient. My intent was to get the old wiki content migrated so that is no longer an outstanding task. Should we create tickets to address or iterate on this PR until it's right?

File: source/frequently-asked-questions/frequently-asked-questions.rst [hunk 3]
```diff
@@ -2,3 +2,333 @@
...
+If you prefer to learn by watching videos, see: 
+
++ `Introducing Ethereum <http://youtu.be/mRWNr3MA2jo>`__ (Video, 10min) 
++ `Vitalik Buterin reveals Ethereum at Bitcoin Miami 2014 <http://youtu.be/l9dpjN3Mwps>`__ (Video, 28min) 
++ `Singularity 1 on 1: Ethereum is a Decentralized Consensus Platform <http://youtu.be/fbEtivJIfIU>`__ (Video, 69min) 
++ `Our second Reddit "Ask Me Anything" for community selected questions <http://www.reddit.com/r/IAmA/comments/2bjmgb/hi_we_are_the_ethereum_project_team_ask_us/>`__ (not actually a video)
+
+Where can I learn more about Ethereum?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```
Comment (zelig) [inline source/frequently-asked-questions/frequently-asked-questions.rst]: this should be a cross ref to community section of the ecosystem. 
Forum will cease to exist. Gitter channels should be listed in detail 
based on https://gitter.im/orgs/ethereum/rooms

**Comment by Souptacular** [comment]
We discussed on Gitter and determined that this PR should be closed since the FAQs transferred were too old. We can reopen if we feel it is necessary.

**Pull request closed without merging**